hngprojects/hng_boilerplate_java_web
# Feat: video to gif conversion

### **Description:**
This pull request introduces a new API endpoint to convert videos to GIFs. The endpoint accepts a video file and triggers a job to perform the conversion.

**Key Features:**

- Endpoint: POST /convert/video-to-gif
- Request Parameter: MultipartFile video (video file to be converted)
- Response: Returns the job status and details upon successful initiation of the conversion process.

**How to Manually Test:**

1. Open Postman (or a similar API testing tool).
2. Set the request type to POST.
3. Enter the endpoint URL: api/v1/convert/video-to-gif.
4. Go to the "Body" tab and select form-data.
5. Add a key named video and set the type to File.
6. Upload a video file you want to convert.
7. Send the request.
8. Verify the response:
- The status should be 201 Created.
- The response body should contain details about the conversion job.
